core/src/main/java/org/apache/struts2/components/UIBean.java

```diff
@@ -689,7 +689,9 @@ public void evaluateParams() {
         }
 
         if (labelPosition != null) {
-            addParameter("labelposition", findString(labelPosition));
+            String labelPosition = findString(this.labelPosition);
+            addParameter("labelposition", labelPosition);
```

Comment:
Not sure but I think not required anymore as all usages in all ftl files are removed.

Reply:
Yet someone can depend on that in his custom theme.